lib/utils/split-files.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * splits a single string into multiple files. The initial content is the root content, the additional splits will
+ * show up within thier own "files" object within the result object returned.
+ * @param content The string to be split
+ * @param regex defaults to a regex that supports `// @config: split-file File.ext`
+ * @returns {{root: string, files: {}, splits: number}}
+ */
+module.exports = function splitFiles(content, regex = /^[ \t#|\/]* ?@config[: ] ?split-file (.*$)/gm) {
```

Comment:
Codewars and Qualified do not yet support opts.files, only CodeStudio. Its just a way for us to convert the solution/setup/fixture format to a multi-file format when absolutely needed. Its mostly needed because Java is picky about file names.